core/src/test/java/io/questdb/griffin/ExplainPlanTest.java

```diff
@@ -3606,6 +3611,50 @@ public void testRewriteAggregateWithAdditionIsDisabledForNonIntegerType() throws
         });
     }
 
+    @Test
+    public void testRewriteAggregateWithAdditionOnJoin() throws Exception {
```

Comment:
does this also work with functions that have 3 or more args?

Reply:
No, this rewrite only applies to simple expressions within select aggregate functions .